src/utils/readtime.utils.js

```diff
@@ -0,0 +1,72 @@
+import moment from 'moment';
+import momentDurationFormatSetup from 'moment-duration-format';
+import faker from 'faker';
+
+momentDurationFormatSetup(moment);
+
+/**
+ * @description counts the number of words in a string.
+ * excluding spaces, tab and newlines
+ * @param {string} words
+ * @returns {number} number of words
+ */
+export function wordCount(words) {
+  return Math.floor(words.split(/[^\s]+/).length - 1);
+}
+
+/**
+ * @description calculate the time to view images in an article
+ * @param {array} images
+ * @returns {number} time in seconds for all images
+ */
+export function calcImageTime(images) {
+  let initSeconds = 11;
+  let totalSeconds = 0;
+  for (let image of images) {
+    totalSeconds += initSeconds;
+    if (initSeconds > 3) {
+      initSeconds -= 3;
+    }
+  }
+  return totalSeconds;
+}
+
+/**
+ * @description calculate the total average time to read videos
+ * @param {array} videos list the various video durations ['1023', '23023']
+ * @returns {number} minutes to view all videos
+ */
+export function calcVideoTime(videos) {
+  let totalDuration = 0;
```

Comment:
`videos` seems to be an array of the duration of each video. Using the variable name 'video' would become confusing. What do you think?

Reply:
Yeah, that's right. I would look into it.